tests/tests.sh

```diff
@@ -11,18 +16,18 @@ oneTimeTearDown() {
 }
 
 test_smoke() {
-  curl --silent http://localhost:$TEST_PORT > /dev/null
+  curl --silent "http://localhost:$TEST_PORT > /dev/null"
```

Comment:
How the heck did you succeed in Jenkins?! It fails for me locally now with an exit code 3, i.e. URL malformed, which is what I would have expected then...